### ISSUE TITLE
missing version info in shredder

### ISSUE THREAD

**Issue by algent-al** [sahib/rmlint#416]
OS: Solus

![shredder](https://user-images.githubusercontent.com/46356616/84182500-c8d78e00-aa8a-11ea-8688-7864a69eb394.png)
Normally instead of `?.?.?` it shoud be `2.10.0`

**Comment by JLuszawski** [comment]
You are right. New version does not match regex used to capture it: _r'version (\d.\d.\d)'_
Should be _r'version (\d+\.\d+\.\d+)'_
